Routes/backend/profile.php

```diff
@@ -7,22 +7,22 @@
     ],
     function () {
         //---------------------------------------------------------
-        Route::post('/', 'UsersController@getProfile')
+        Route::post('/', 'ProfileController@getItem')
             ->name('backend.vaah.profile');
         //---------------------------------------------------------
-        Route::post('/assets', 'UsersController@getAssets')
+        Route::post('/assets', 'ProfileController@getAssets')
             ->name('backend.vaah.profile.assets');
         //---------------------------------------------------------
-        Route::post('/store', 'UsersController@storeProfile')
+        Route::post('/store', 'ProfileController@storeItem')
             ->name('backend.vaah.profile.store');
         //---------------------------------------------------------
-        Route::post('/store/password', 'UsersController@storeProfilePassword')
+        Route::post('/store/password', 'ProfileController@storePassword')
             ->name('backend.vaah.profile.store.password');
         //---------------------------------------------------------
-        Route::post('/avatar/store', 'UsersController@storeProfileAvatar')
+        Route::post('/avatar/store', 'ProfileController@storeAvatar')
             ->name('backend.vaah.profile.avatar.store');
         //---------------------------------------------------------
-        Route::post('/avatar/remove', 'UsersController@removeProfileAvatar')
+        Route::post('/avatar/remove', 'ProfileController@removeAvatar')
```

Comment:
@we-vikram-wri231 we should start using `Route::get('/user', [UserController::class, 'index']);` format, this will help to open the controller quickly. Read more at https://laravel.com/docs/10.x/routing#the-default-route-files, we should make this change to CRUD also.